austation/code/modules/mob/living/human/species_type/apid.dm

```diff
@@ -0,0 +1,2 @@
+/datum/species/apid/after_equip_job(datum/job/J, mob/living/carbon/human/H)
+	H.grant_language(/datum/language/buzzwords)
```

Comment:
you need to add `. = ..()` to the start of all these after_equip_job procs else it'll break stuff because the old definition isn't getting called too.